src/parser/ast/type_argument_term.rs

```diff
@@ -0,0 +1,42 @@
+use crate::ir::ir_component::IRComponent;
+use crate::parser::ast::traits::ast_term::ASTTerm;
+
+pub struct TypeArgumentTerm {
```

Comment:
Could we use a word dimension instead argument to describe the sense of this term?